aws_instance.tf

```diff
@@ -5,8 +5,13 @@
 resource "aws_instance" "instanceWithNoVpc" {
   ami           = "some-id"
   instance_type = "t2.micro"
-  
+
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Instances are not configured under custom Virtual Private Cloud**
 
**Vulnerability**
By default instances are launched in default VPC which is not recommended for production grade. AWS VPCs provides the controls to facilitate a formal process for approving and testing all network connections and changes to the firewall and router configurations.
 
**Remediation**
Migrate from EC2 classic to VPCs. 1: Create a VPC. 2: Configure Your Security Group. 3: Create an AMI from Your EC2-Classic Instance. 4: Launch an Instance Into Your VPC. Detailed steps available at:Â http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/vpc-migrate.html

source: accurics-